Updates/0333_dbscript_string_broadcast_text_id.sql

```diff
@@ -0,0 +1,200 @@
+# ################################################################################
+# TABLE dbscript_string
+# ################################################################################
+
+# --------------------------------------------------------------------------------
+# ## tab 2000005032	=> "...And then the Tauren said, "13 INCHES!""
+# -- mal [6405] "And then the Tauren said, '13 INCHES!'"
+
+UPDATE dbscript_string SET broadcast_text_id = 6405 WHERE entry = 2000005032;
+
+# --------------------------------------------------------------------------------
+# ## tab 2000000215	=> "Apothecary Zamah smiles."
+# NONE FOUND
+
```

Comment:
Could be
UPDATE dbscript_string SET broadcast_text_id = 1418 WHERE entry = 2000000215;

Reply:
https://github.com/cmangos/tbc-db/blob/master/Updates/0859_dbscript_string_broadcast_text_update.sql#L55

dont bother too much alot is already done.